yb-voyager/src/srcdb/postgres.go

```diff
@@ -1116,13 +1107,88 @@ func (pg *PostgreSQL) GetMissingExportDataPermissions(exportType string) ([]stri
 			return nil, fmt.Errorf("error in checking table owner permissions: %w", err)
 		}
 		if len(missingTables) > 0 {
-			combinedResult = append(combinedResult, fmt.Sprintf("Tables over which user does not have ownership: %v", missingTables))
+			combinedResult = append(combinedResult, fmt.Sprintf("\n%s[%s]", color.RedString("Missing ownership for user %s on Tables: ", pg.source.User), strings.Join(missingTables, ", ")))
+		}
+
+		// Check if sequences have SELECT permission
+		sequencesWithMissingPerm, err := pg.listSequencesMissingSelectPermission()
+		if err != nil {
+			return nil, fmt.Errorf("error in checking sequence select permissions: %w", err)
+		}
+		if len(sequencesWithMissingPerm) > 0 {
+			combinedResult = append(combinedResult, fmt.Sprintf("\n%s[%s]", color.RedString("Missing SELECT permission for user %s on Sequences: ", pg.source.User), strings.Join(sequencesWithMissingPerm, ", ")))
+		}
+	} else {
+		// For offline migration
```

Comment:
shouldn't we be checking GetMissingExportSchemaPermissions even for live migration? 
live migration checks are a superset of offline migration checks.

Reply:
Yeah we do check them internally in the GetMissingExportDataPermissions function

Reply:
Ah okay, we don't check for listTablesMissingSelectPermission, because we are anyway checking for ownership, correct?

Reply:
Yep